server/lib/statsd-collector.js

```diff
@@ -65,10 +75,34 @@ function getGenericTags(body) {
     }
   }
 
+  // collect AB choices
+  if (body.ab) {
+    var abTags = [];
+
+    body.ab.forEach(function (item) {
+      if (item.event && item.event === 'choice' && item.experiment && item.choice) {
+        var tagName = 'ab_' + _normalizeTag(item.experiment);
+        var hadChoice = false;
+
+        Object.keys(item.choice).forEach(function (choiceKey) {
+          hadChoice = true;
+          tagName += '_' + _normalizeTag(choiceKey);
```

Comment:
Using an `_` here makes it difficult to separate the choice key from the rest of the name, since spaces are replaced with `_`. Maybe another `:` here, or maybe `:` should be the universal field separator?